examples/quickstarts/helidon-quickstart-se/pom.xml

```diff
@@ -106,6 +106,20 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.graalvm.buildtools</groupId>
+                <artifactId>native-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>build-native-image</id>
+<!--                        <configuration>-->
```

Comment:
Why is this commented out ?

Reply:
I think I left in some of my intermediate work. Will check this